File: rb/utility/make_context.rb
# BluefinTecsMerchantPortal SDK utility: make_context
require_relative '../core/context'
module BluefinTecsMerchantPortalUtilities
  MakeContext = ->(ctxmap, basectx) {
    BluefinTecsMerchantPortalContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// BluefinTecsMerchantPortal SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class BluefinTecsMerchantPortalMakeContext
{
    public static function call(array $ctxmap, ?BluefinTecsMerchantPortalContext $basectx): BluefinTecsMerchantPortalContext
    {
        return new BluefinTecsMerchantPortalContext($ctxmap, $basectx);
    }
}
